docs/user-guide/federation/federated-ingress.md

```diff
@@ -157,6 +157,17 @@ the network traffic directed to this ingress (i.e. 'Service
 Endpoints' behind the service backing the Ingress), so the Federated Ingress does not yet consider these to
 be healthy shards and will not direct traffic to any of these clusters.
 
+Thirdly, it is worth noting that the federation control system will
```

Comment:
Thirdly? I don't see Firstly or Secondly, and we probably want to avoid that phrasing anyway.

"Note that the federation control system..."